### PR TITLE
Feature/css cleanup

### PR DESCRIPTION
**Pull request by itsnameissteven**
## What does this PR do?
* Adds mobile and tablet view to app. 
* Cleans up small css details
* Animated arrow on side bar
* New color scheme
-

## How should it be tested in the UI?
open up in browser and test for responsiveness. 
-

### CODE REVIEW

File: src/components/SideBar/SideBar.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import './SideBar.css'
-import { GiHamburgerMenu } from "react-icons/gi"
+import { HiArrowRight } from "react-icons/hi"
```
Comment (Jacksonmcguire) [inline src/components/SideBar/SideBar.tsx:3]: Good call on switching to an arrow here, hamburger seemed out of place

**Review by cbmackintosh — APPROVED** [review]
Everything looks good to me. I'll incorporate the new color palette into my error message styling on my next PR.

File: src/components/SideBar/SideBar.tsx [hunk 2]
```diff
@@ -10,14 +10,18 @@ type ApiProp = {
...
+  }
 
   return (
     <div className='side-bar'>
-      <GiHamburgerMenu 
-      className="hamburger"
-      onClick={() => document.querySelector('.side-bar')?.classList.toggle('open')}></GiHamburgerMenu>
+      <HiArrowRight 
+      className="arrow"
+      onClick={toggleSideBar} />
```
Comment (Jacksonmcguire) [inline src/components/SideBar/SideBar.tsx:22]: Good call breaking out this event listener to its own function to get rid of return functionality

**Review by Jacksonmcguire — APPROVED** [review]
Looks good to go, shouldn't bring up any merge conflicts on my end

**Pull request merged**